api/handlers.go

```diff
@@ -359,6 +383,18 @@ func getVolumes(c *context, w http.ResponseWriter, r *http.Request) {
 			continue
 		}
 
+		found = false
+		for _, driver := range drivers {
+			if strings.Contains(volume.Driver, driver) {
```

Comment:
The daemon code uses `filter.ExactMatch` instead of a substring match here:

https://github.com/moby/moby/blob/a3efe9722f34af5cf4443fe3a5c4e4e3e0457b54/daemon/list.go#L636

Reply:
makes sense, I will correct that

Reply:
...done